webrtc/go/manifest.toml

```diff
@@ -0,0 +1,22 @@
+name = "webrtc:go"
+
+[builders]
+"exec:go"   = { enabled = true }
+
+[builders."docker:go"]
+enabled                = true
+build_base_image       = "golang:1.19-bullseye"
+
+[runners]
+"local:docker"  = { enabled = true }
+"local:exec"    = { enabled = true }
+"cluster:k8s"   = { enabled = true }
+
+[[testcases]]
+name = "ping"
+instances = { min = 1, max = 10000, default = 1 }
```

Comment:
why a max of `10000`? How do you ever expect such numbers to be reached?

Reply:
The thing I changed here was to lower the default.

I would guess if someone asked for 10000 they're actually expecting the system to break and want to see what happens when it does. Though I don't really know.

Reply:
Hmmm ok. Seems odd, but plausible.